### PR TITLE
[bitwarden_rs] New chart

### PR DESCRIPTION
**Pull request by DirtyCajunRice**
Minimal chart addition. 
### Future Plans
* [x] Change env vars to configmap to allow for easier helm management of statefulsets
* Encompass all possible configuration options - after first merge

### CODE REVIEW

**Comment by billimek** [comment]
Can we please globally rename `bitwarden_rs` to `bitwarden-rs`?  The underscores (`_`) break things and the 'standard' seems to be hyphens (`-`) based on what I see over in [the stable charts repo](https://github.com/helm/charts/tree/master/stable/)

File: charts/bitwardenrs/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+This is an opinionated helm chart for [bitwarden_rs](https://github.com/dani-garcia/bitwarden_rs) 
+
+The default values and container images used in this chart will allow for running in a multi-arch cluster (amd64, arm, arm64)
+
+## TL;DR
+
+```console
+helm repo add billimek https://billimek.com/billimek-charts/
+helm install billimek/bitwarden_rs
```
Comment (onedr0p) [inline charts/bitwardenrs/README.md:11]: Need to update these names, bitwardenrs, right?

File: charts/bitwardenrs/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+## Installing the Chart
+
+To install the chart with the release name `bitwarden`:
+
+```console
+helm install bitwarden billimek/bitwarden_rs
```
Comment (onedr0p) [inline charts/bitwardenrs/README.md:19]: and here

File: charts/bitwardenrs/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Configuration
+
+Read through the [values.yaml](https://github.com/billimek/billimek-charts/blob/master/charts/bitwarden_rs/values.yaml) file. It has several commented out suggested values.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+helm install bitwarden \
+  --set timeZone="America/New York" \
+    billimek/bitwarden_rs
```
Comment (onedr0p) [inline charts/bitwardenrs/README.md:41]: and here

File: charts/bitwardenrs/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+helm install bitwarden billimek/bitwarden_rs  --values values.yaml
```
Comment (onedr0p) [inline charts/bitwardenrs/README.md:47]: and here

File: charts/bitwardenrs/templates/secret.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{{- if not .Values.bitwardenrs.admin.existingSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "bitwardenrs.fullname" . }}
+  labels:
+  {{- include "bitwardenrs.labels" . | nindent 4 }}
+type: Opaque
+data:
+  {{- if not .Values.bitwardenrs.admin.existingSecret }}
```
Comment (onedr0p) [inline charts/bitwardenrs/templates/secret.yaml:10]: this is the same logic that's on line 1

Comment (DirtyCajunRice) [reply charts/bitwardenrs/templates/secret.yaml]: fixed good catch

File: charts/bitwardenrs/README.md (hunk 4 above)
Comment (DirtyCajunRice) [reply charts/bitwardenrs/README.md]: fixed

File: charts/bitwardenrs/README.md (hunk 3 above)
Comment (DirtyCajunRice) [reply charts/bitwardenrs/README.md]: fixed

File: charts/bitwardenrs/README.md (hunk 2 above)
Comment (DirtyCajunRice) [reply charts/bitwardenrs/README.md]: fixed

File: charts/bitwardenrs/README.md (hunk 1 above)
Comment (DirtyCajunRice) [reply charts/bitwardenrs/README.md]: fixed

**Comment by onedr0p** [comment]
lgtm!

**Pull request merged**